settings.gradle

```diff
@@ -1,2 +1,9 @@
-rootProject.name = 'omero-api-plugin'
+pluginManagement {
+    repositories {
+        mavenLocal()
+        gradlePluginPortal()
```

Comment:
That means ``org.openmicroscopy.plugin-project`` is only found from local repo.
This will not be found from our artifactory

Reply:
Resurrecting artifactory in all of these blocks for the moment. (At least until plugins are pushed elsewhere)